gc/base/GCExtensionsBase.cpp

```diff
@@ -356,3 +357,9 @@ MM_GCExtensionsBase::usingSATBBarrier()
 #endif /* defined(OMR_GC_REALTIME) */
 	return false;
 }
+
+bool
+MM_GCExtensionsBase::isAddressWithinHeap(void *address)
```

Comment:
let's try without introducing this method. instead, we can do:
- simplest from testing perspective, but probably least desirable, introduce it on J9 side
- use (NULL != regionManager->regionDescriptorForAddress())  instead, which probably also a reasonable simple compromise
- examine the caller sites more carefully and see if it's really need at all, or if we can use isDataAddr(Non)Adjacent instead